dsls/sql/src/test/java/org/apache/beam/dsls/sql/planner/BeamPlannerAggregationSubmitTest.java

```diff
@@ -41,10 +42,11 @@
   public static DateFormat format = new SimpleDateFormat("yyyy-MM-dd HH:mm:ss");
   public static BeamSqlRunner runner = new BeamSqlRunner();
 
-  @BeforeClass
```

Comment:
Any thoughts to replace with `Before`? I suppose the `prepare()` only need to run once, which create the initial tables for all test cases.